src/Qowaiv/Mathematics/DecimalMath.cs

```diff
@@ -159,4 +160,7 @@ private static Random Random()
 
     [ThreadStatic]
     private static Random? _rnd;
+#else
+    private static Random Random() => System.Random.Shared;
```

Comment:
Perhaps a good idea to add a comment here noting that `System.Random.Shared` is thread-safe

Reply:
Added.